xml_converter/tests/attribute_testing.py

```diff
@@ -0,0 +1,140 @@
+from dataclasses import dataclass
+import subprocess
+import re
+import os
+from typing import Dict, List
+
+# Path to compiled C++ executable
+xml_converter: str = "../build/xml_converter"
+
+
+@dataclass
+class Attribute_Tests:
+    attribute_name: str
+    attribute_path: str
+    tests: List[str]
+    output_result: Dict[str, List[str]]
+    expect_output_to_be_equal: Dict[str, bool]
+    expect_files_to_be_equal: Dict[str, bool]
+
+
+def run_xml_converter(*args):
```

Comment:
this immediately gets converted to a list, just use a list

Reply:
Done. Changed to reflect the 5 arguments that xml_converter could take in 

```
def run_xml_converter(
    input_xml: Optional[List[str]]= None, 
    output_xml: Optional[List[str]]= None,
    input_proto: Optional[List[str]]= None,
    output_proto: Optional[List[str]]= None,
    split_output_proto: Optional[str]= None,
    ) 
```